### PR TITLE
Updated readable-proxy use to fetch sanitized contents.

### PR DESCRIPTION
**Pull request by n1k0**

### CODE REVIEW

**Comment by n1k0** [comment]
@ametaireau r=?

**Comment by Natim** [comment]
:+1:

File: src/test/content_test.js [hunk 1]
```diff
@@ -43,6 +43,14 @@ describe("ContentManager", function() {
       sinon.assert.calledWithMatch(manager.client, {params: {url: article.url}});
     });
 
+    it("should fetch natitized contents from the API", function() {
```
Comment (almet) [inline src/test/content_test.js]: sanitized

**Comment by almet** [comment]
WFM with the small nitpick.

**Pull request merged**